### DISCUSSION TITLE
OpenCV and CustomTkinter for webcam view

### DISCUSSION THREAD

**Discussion by HeyImAlice** [TomSchimansky/CustomTkinter#720] [General]
Hi !

I'm actually trying to create an app with a frame for a webcam flux in it.
I find some solution for Tkinter, but not for customTkinter.

If you can help me, I appreciate :D

thanks !

**Comment by sumeshir26** [comment]
Try using it exacly as you would use in Tkinter except with thr Ctk widgets

**Comment by HeyImAlice** [comment]
yeah it finally work

I just forget a self. ...

Thanks XD

**Reply by corgan2222** [reply]
Can you please share, how you did it? Trying the same. Thanks in advance

**Reply by HeyImAlice** [reply]
yes of course, I used this :
https://github.com/andikaMaulana/Python-OpenCV-webcam-capture/blob/master/webcamGUI.py
